src/state/pymMiddleware.js

```diff
@@ -0,0 +1,13 @@
+import pym from 'pym.js';
+
+const pymChild = new pym.Child();
+
+/**
+ * Update pym each time an action is fired.
+ */
+const pymMiddleware = () => next => (action) => {
+  pymChild.sendHeight();
+  return next(action);
+};
```

Comment:
We're not exactly a real-time app, but there's a few places where a number of actions will fire in sequence in a short period of time, such as when a filter action is taken for which new data is requested -- would there be any performance concern to doing this on every update, and/or should we consider de-bouncing this? I ask this naively because I am unfamiliar with the overhead of running this library.